components/calendar/src/cal/hijri.rs

```diff
@@ -49,6 +49,16 @@ mod ummalqura_data;
 ///
 /// This calendar is a pure lunar calendar with no leap months. It uses month codes
 /// `"M01" - "M12"`.
+///
+/// # Precise definition and limits
+///
+/// This calendar generically covers any pure lunar calendar used liturgically in Islam,
+/// with 12 months each of length 29 or 30, with an epoch intended to mark the Hijrah in 622 CE.
```

Comment:
Note: I'm fine saying that the epoch is the Hijrah, but I thought we previously said we needn't make that commitment, which is why we added ECMA reference year functions to the hijri::Rules trait.

Reply:
I think from a calendrical definition perspective saying the epoch is Hijrah is probably good.

Reply:
we need that actually, because `PackedHijriYearData` needs it